src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -1534,25 +1534,6 @@ public void BuildApplicationCheckThatAddStaticResourcesTargetDoesNotRerun ()
 			}
 		}
 
-		[Test]
-		[Ignore ("Re enable when MergeResources work is complete")]
```

Comment:
Future work. Let's remove all of the tests labeled with this. As well as the code for the managed resource merger. I think we are just going to use the google tooling (aapt2) going forward.